examples/geometry_optimization.jl

```diff
@@ -52,33 +57,32 @@ function fg!(F, G, x)
         G .= -[grad[1][1]; grad[1][2]]
     end
     scfres.energies.total
-end
+end;
 
 # Now, we can optimize on the 6 parameters `x = [x1, y1, z1, x2, y2, z2]` in
-# reduced coordinates, using `LBFGS()` and starting from `x0`. By
+# reduced coordinates, using `LBFGS()`, the default minimization algorithm
+# in Optim. We start from `x0`, which is a first guess for the coordinates. By
 # default, `optimize` traces the output of the optimization algorithm during the
 # iterations. Once we have the minimizer `xmin`, we compute the bond length in
 # cartesian coordinates.
 
-x0 = vcat(invlattice*[0., 0., 0.], invlattice*[1.4, 0., 0.])
+x0 = vcat(lattice \ [0., 0., 0.], lattice \ [1.4, 0., 0.])
 xres = optimize(Optim.only_fg!(fg!), x0, LBFGS(),
                 Optim.Options(show_trace=true, f_tol=tol))
 xmin = Optim.minimizer(xres)
 dmin = norm(lattice*xmin[1:3] - lattice*xmin[4:6])
-sdmin = @sprintf "%.3f" dmin
-println("\nBond length minimizing the energy for Ecut=$(Ecut): ", sdmin, " Bohr");
+@printf "\nOptimal bond length for Ecut=%.2f: %.3f Bohr\n" Ecut dmin
 
-# We used here very rough parameters to generate the example,
-# but setting `Ecut` to 10 Ha yields a bond length of 1.523 Bohr.
-# As a comparison, ABINIT with the exact same lattice constant, Ecut = 10,
+# We used here very rough parameters to generate the example and
+# setting `Ecut` to 10 Ha yields a bond length of 1.523 Bohr.
+# As a comparison, ABINIT with the exact same lattice constant, `Ecut = 10`,
```

Comment:
Typo here: ABINIT yields the same bond length

Reply:
Do we mention that the parameters on ABINIT are the same or do we just write "ABINIT yields the same bond length" ?

Reply:
Maybe just add `, which [agrees with ABINIT](link to tutorial).` to the first sentence and leave the rest. It's implicitly clear that the same setup is used.

Reply:
yeah, sounds good :+1: